### ISSUE TITLE
Use msg more in documentation code snippets

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/website-v2#325]
For example, https://scratchaddons.com/docs/develop/userscripts/best-practices/ has the following code snippet:
```js
// Do this instead:
const span = document.createElement("span");
span.textContent = `Remix ${projectTitle}`;
document.querySelector(".sa-remix-button").append(span);
```
Which probably should be:
```js
// Do this instead:
const span = document.createElement("span");
span.textContent = msg("remix-button", { projectTitle });
document.querySelector(".sa-remix-button").append(span);
```

**Comment by WorldLanguages** [comment]
That specific example is about XSS, so using `msg()` there can make it confusing. It becomes more complicated to explain because `safeMsg` would also be worth mentioning. Also, the reader might incorrectly interpret that `msg()` avoids XSS, instead of `el.textContent`.

I guess we could add a code comment such as: `// Note: real userscript code should use msg() here`

**Comment by mybearworld** [comment]
> That specific example is about XSS, so using `msg()` there can make it confusing. It becomes more complicated to explain because `safeMsg` would also be worth mentioning. Also, the reader might incorrectly interpret that `msg()` avoids XSS, instead of `el.textContent`.
> 
> I guess we could add a code comment such as: `// Note: real userscript code should use msg() here`

True, that specific example was poorly chosen, I'll replace it with another one.

**Comment by WorldLanguages** [comment]
I believe we should keep the current examples, but we could add a comment such as `// Note: real userscript code should use msg() here` next to the lines that use hardcoded strings.
